src/strategy-management/Toolkit.ts

```diff
@@ -8,7 +8,6 @@ import {
   tenPow,
   formatUnits,
   parseUnits,
-  trimDecimal,
```

Comment:
If this function is no longer used **anywhere**, then you may also remove its implementation from the codebase.

Reply:
it is used in one more place